### PR TITLE
feat: add additional exercises and solutions to TS workshop

### PR DESCRIPTION
**Pull request by AshNaz87**
Feel free to edit and change as you wish

### CODE REVIEW

File: typescript/types/src/assigning-types/assigning-types-problem.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * Create an interface for a footballer. This footballer has some properties. Using the information below, create a type alias or an interface:
+ *
+ * - Name => "Goal Hanger"
+ * - Age => 30
+ * - Football Club => "Norwich City F.C"
+ * - Position => "Striker"
+ * - Shirt number => 9
+ * - Salary => 1000000
+ *
+ * The type or an interface should be generic.
```
Comment (AshNaz87) [inline typescript/types/src/assigning-types/assigning-types-problem.ts]:
```suggestion
 * The type or an interface should be generic. Not to be confused with generic types!
```

File: typescript/types/src/unions/unions-solution.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+interface MultiSkilledFootballer {
+  name: string;
+  age: number;
+  footballClub: string;
+  position: Position | [Position, Position];
+  shirtNumber: number;
+  salary: number;
+}
+
```
Comment (AshNaz87) [inline typescript/types/src/unions/unions-solution.ts]:
```suggestion

/**
 *
 * @param {MultiSkilledFootballer} footballer
 * @returns {string} footballer's name and position/s
 */
 
```

**Comment by kial91** [comment]
@AshNaz87 I think you might have added the exercises at the wrong level. Union types and assigning types are topics within themselves (see slide deck). These should be their own folders.

As for basic numbers - maybe that should just be within the index.js of types?

**Comment by AshNaz87** [comment]
Happy for you to choose the file structure

**Review by kial91 — APPROVED** [review]

**Pull request merged**